nsv13/code/game/objects/items/bloody_eye.dm

```diff
@@ -0,0 +1,194 @@
+	M.Jitter(20)
+	if(prob(20))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "laugh"))
+	..()
+
+/datum/reagent/drug/bloody_eye/addiction_act_stage2(mob/living/M)
+	M.Jitter(50)
+	M.Dizzy(5)
+	if(prob(30))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "laugh"))
+	..()
+
+/datum/reagent/drug/bloody_eye/addiction_act_stage3(mob/living/M)
+	if((M.mobility_flags & MOBILITY_MOVE) && !ismovableatom(M.loc))
+		for(var/i = 0, i < 4, i++)
+			moveRange(M, pick(GLOB.cardinals))
+	M.Jitter(100)
+	M.Dizzy(10)
+	if(prob(40))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "gasp"))
+	..()
+
+/datum/reagent/drug/bloody_eye/addiction_act_stage4(mob/living/carbon/human/M)
+	if((M.mobility_flags & MOBILITY_MOVE) && !ismovableatom(M.loc))
+		for(var/i = 0, i < 6, i++)
+			step(M, pick(GLOB.cardinals))
+	M.Jitter(150)
+	M.Dizzy(15)
+	M.adjustToxLoss(0.25, 0)
+	if(prob(50))
+		INVOKE_ASYNC(M, /mob.proc/emote, pick("scream", "laugh"))
+	..()
+	. = 1
+
+/obj/item/reagent_containers/hypospray/bloody_eye/attack(mob/living/M, mob/user)
+	if(!reagents.total_volume)
+		to_chat(user, "<span class='warning'>[src] is empty!</span>")
+		return
+	if(!iscarbon(M))
+		return
+
+	var/list/injected = list()
+	for(var/datum/reagent/R as() in reagents.reagent_list)
+		injected += R.name
+	var/contained = english_list(injected)
+	log_combat(user, M, "attempted to spray", src, "([contained])")
+
+	if(reagents.total_volume && !(M.is_eyes_covered()))
+		to_chat(M, "<span class='warning'>Your eye is covered in a fine red mist!</span>")
+		to_chat(user, "<span class='notice'>You spray [M]'s eye with [src].</span>")
+		playsound(loc, 'sound/items/hypospray.ogg', 50, 1)
+
+		var/fraction = min(amount_per_transfer_from_this/reagents.total_volume, 1)
+		reagents.reaction(M, INJECT, fraction)
+		if(M.reagents)
+			var/trans = 0
+			if(!infinite)
+				trans = reagents.trans_to(M, amount_per_transfer_from_this, transfered_by = user)
+			else
+				trans = reagents.copy_to(M, amount_per_transfer_from_this)
+
+			to_chat(user, "<span class='notice'>[trans] unit\s sprayed.  [reagents.total_volume] unit\s remaining in [src].</span>")
+
+
+			log_combat(user, M, "sprayed", src, "([contained])")
+	else
+		to_chat(M, "<span class='warning'>You have to uncover the eyes first!</span>")
+
+/datum/uplink_item/dangerous/bloody_eye
+	 name = "Bloody Eye Spray"
+	 desc = "Spray into your eye or someone else's to go into a bloody thirsty rage. Don't expect to live after using it."
+	 item = /obj/item/reagent_containers/hypospray/bloody_eye
+	 cost = 10
+
+/datum/status_effect/bloody_eye
+	id = "bloody_eye"
+	duration = -1
+	status_type = STATUS_EFFECT_UNIQUE
+	tick_interval = 1
+	examine_text = "<span class='warning'>Their eyes are blood red!</span>"
+	alert_type = null
+
+/datum/status_effect/bloody_eye/tick()
+	owner.a_intent_change(INTENT_HARM)
+	if(owner.a_intent != INTENT_HARM)
+		owner.a_intent_change(INTENT_HARM)
+		to_chat(owner, "<span class='warning'>KILLKILLKILL</span>")
```

Comment:
doesn't kill

Reply:
I don't know what you mean by this but the point is that the drug puts a player into a bloodthirsty rage. And so if they try to change intent to something else, it'll send that message telling them to kill.

Reply:
It's because github cut my comment.
You are setting the intent to harm, you don't have to check if it's harm

Reply:
Gotcha.